robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAccessibilityEvent.java

```diff
@@ -228,4 +267,22 @@ public int hashCode() {
       return mEvent.hashCode();
     }
   }
+
+  @Implementation
+  public int describeContents() {
+    return 0;
+  }
+
+  @Implementation
+  public void writeToParcel(Parcel dest, int flags) {
+    StrictEqualityEventWrapper wrapper = new StrictEqualityEventWrapper(realAccessibilityEvent);
+    int keyOfWrapper = -1;
+    for (int i = 0; i < orderedInstances.size(); i++) {
+      if (orderedInstances.valueAt(i).equals(wrapper)) {
+        keyOfWrapper = orderedInstances.keyAt(i);
+        break;
+      }
+    }
+    dest.writeInt(keyOfWrapper);
+  }
```

Comment:
If we're going to do this 100%, we should really extend ShadowAccessibilityRecord along with all of its getters and setters.